exercises/input.js

```diff
@@ -69,17 +109,28 @@ class Input{
 		MAKE SURE TO STORE the reference to the dom element in the object for later use!
 		Don't store the CSS selector, you made the element, store the direct dom object itself!
 		*/
-	showError(  ){
-
+	showError( message ){
+		var positionInput = this.testInput.position();
+        var heightInput = this.testInput.height();
+        var parentInput = this.testInput.parent();
+        this.errorMessage = $("<div>",{
+			css: {
+				top: positionInput.top + heightInput,
+				left: positionInput.left
+			},
+			"class": "inputError",
+            text: message
+		});
+		parentInput.append(this.errorMessage);
```

Comment:
Also may want to remove the error message if it was already there so you don't get duplicates